serverlessFirstSteps/Easily-hosting-your-portfolio-online/hosting-a-front-end-website-building-portfolio.md

```diff
@@ -1,4 +1,4 @@
-# Easily hosting your portfolio online: an introduction to CodeCapsules.io
+# Hosting a front-end website: building and hosting your professional portfolio
 
 Deploying a web application to a production environment is complex. Traditionally you set up a server and micromanage the tasks associated with hosting one: choosing the servers' operating system, editing its configurations, and monitoring the server to ensure it is running.
```

Comment:
This isn't a web application - just a web page. I think the compelling part here is that as a developer building your portfolio is fun but 
* Figuring out how to host it is hard
* Updating it x-years later and remembering how to build and deploy is a pain - you want a set up where you can just make changes and git push, even 5 years later
* Worry about the code, not the hosting and deployment